pinot-common/src/test/java/org/apache/pinot/common/utils/FileUtilsTest.java

```diff
@@ -0,0 +1,125 @@
+package org.apache.pinot.common.utils;
+
+import com.google.common.collect.Iterables;
+import junit.framework.TestCase;
+import org.junit.jupiter.api.Test;
+
+import java.io.Closeable;
+import java.io.File;
+import java.io.IOException;
+import java.io.RandomAccessFile;
+import java.nio.channels.FileChannel;
+import java.util.HashMap;
+import java.util.Map;
+
+public class FileUtilsTest extends TestCase {
```

Comment:
Add javadoc to point to the class being tested, and perhaps add details on what is being tested.